ethcore/src/miner/miner.rs

```diff
@@ -1451,8 +1454,9 @@ impl miner::MinerService for Miner {
 						service_transaction_checker.as_ref(),
 					);
 					queue.cull(client);
-					if is_internal_import {
```

Comment:
was this previously true only for instant seal engine? otherwise, should it be something like
```rust
				if engine.should_reseal_on_update() {
						// force update_sealing here to skip `reseal_required` checks
						chain.update_sealing(ForceUpdateSealing::Yes);
				} else if is_internal_import {
						chain.update_sealing(ForceUpdateSealing::No);
				}
```

Reply:
yeah it was only for `InstantSeal`.